src/database/migrations/0012-create-platform_provider_responses.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE platform_provider_responses (
+  external_id VARCHAR NOT NULL,
+  platform_provider VARCHAR NOT NULL,
+  PRIMARY KEY (external_id, platform_provider),
+  data JSONB NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW()
+);
+
+COMMENT ON TABLE platform_provider_responses IS
+  'Data loaded from data platform providers.';
+COMMENT ON COLUMN applicants.external_id IS
+  'A normalized external identifier based on the value stored in the data platform provider.';
```

Comment:
This comment is no longer accurate in light of our decision to relax the normalization rules, right?

And, with that relevant information no longer being true, I don't think either comment is adding any new information that is not already communicated by the table and column names, so can we delete them?